stats/statsview.cpp

```diff
@@ -55,6 +56,30 @@ StatsView::~StatsView()
 {
 }
 
+void StatsView::mousePressEvent(QMouseEvent *event)
+{
+	// Currently, we only support dragging of the legend. If other objects
+	// should be made draggable, this needs to be generalized.
```

Comment:
that works well on the desktop - I need to test this on mobile. I think drag behaves the same, but I have learned to assume nothing...

Reply:
Very odd effect on iOS (this seemed best to test on a tablet, and that's the only working tablet I have).
Dragging works reasonably well in the `y` direction, but in `x` direction it stops after a few dozen pixels. You can lift the finger and start again, but something is screwy with this :-)